src/Message/CreateSingleUseCardTokenRequest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * PaywayRest Create Single Use Card Token Request
+ */
+namespace Omnipay\PaywayRest\Message;
+
+/**
+ * PaywayRest Create Single Use Card Token Request
+ *
+ * @link https://www.payway.com.au/docs/rest.html#tokenise-a-credit-card
+ */
+class CreateSingleUseCardTokenRequest extends AbstractRequest
+{
+    public function getData()
+    {
+        $this->getCard()->validate();
+
+        // PayWay requires two digit expiry month.
+        $expiryDateMonth = $this->getCard()->getExpiryMonth() > 9
```

Comment:
You can use str_pad to make it a little cleaner.

```php
$expiryDateMonth = str_pad($this->getCard()->getExpiryMonth(), 2, 0, STR_PAD_LEFT);
```

Reply:
Thanks for the review @joshbenham i will have a look and update it.